source/classes.tex

```diff
@@ -4472,8 +4472,8 @@
 \indextext{access control!\idxcode{protected}}%
 protected;
 that is, its name can be used only by members and friends
-of the class in which it is declared, by classes derived from that class, and by their
-friends (see~\ref{class.protected}).
+of the class in which it is declared, and members of classes derived from that class
```

Comment:
"... and by members of classes derived..." (add "by" to keep the sentence structure manageable)